04-store-api/starter/app.js

```diff
@@ -1 +1,35 @@
-console.log('04 Store API')
+require('dotenv').config()
+require('express-async-errors')
+
+const express =  require ('express')
+const app = express()
+
+const connectDB = require('./db/connect')
+const productsRouter = require('./routes/products')
+
+const notFoundMiddleware = require('./middleware/not-found')
+const errorMiddleware = require('./middleware/error-handler')
+
+app.use(express.json())
+
+app.get('/', (req,res)=>{
+    res.send('<h1>Store API</h1><a href="/api/v1/products">products route</a>')
+})
+
+app.use('/api/v1/products',productsRouter)
+
+app.use(notFoundMiddleware)
+app.use(errorMiddleware)
+
+const port =  process.env.PORT || 3000
+
+const start = async () =>{
+    try {
+        await connectDB(process.env.MONGO_URI)
+        app.listen(port, console.log(`Server is listening port ${port}....`))
```

Comment:
Same comment here as this one from a few weeks ago: https://github.com/mautoranzo/node-express-course/pull/5#discussion_r1027394640